SWLOR.Game.Server/Feature/AbilityDefinition/Force/ForceSparkAbilityDefinition.cs

```diff
@@ -67,16 +68,23 @@ private void ForceSpark1()
             _builder.Create(FeatType.ForceSpark1, PerkType.ForceSpark)
                 .Name("Force Spark I")
                 .Level(1)
-                .HasRecastDelay(RecastGroup.ForceSpark, 20f)
-                .RequirementFP(1)
+                .HasRecastDelay(RecastGroup.ForceSpark, 6f)
+                .HasActivationDelay(2f)
                 .IsCastedAbility()
                 .HasMaxRange(10f)
                 .IsHostileAbility()
                 .UsesAnimation(Animation.LoopingConjure1)
                 .DisplaysVisualEffectWhenActivating()
                 .HasImpactAction((activator, target, level, location) =>
                 {
-                    Impact(activator, target, 9, 2, 1, Tier1Tag, 8);
+                    var willBonus = GetAbilityModifier(AbilityType.Willpower, activator);
+                    var willDMG = 10 + (willBonus * 2);
+                    if (Stat.GetCurrentFP(activator) == 0)
```

Comment:
Please change this condition to <= 0. It should be equivalent but if there's ever a bug in the future which results in a negative FP amount, this will catch it.